src/EventTypes/EventTypes.ts

```diff
@@ -0,0 +1,60 @@
+
+
+export module EventTypes {
+
+    /**
+    * @typedef {String} SOLIDITY_EVENT
+    */
+    export const SOLIDITY_EVENT = 'SOLIDITY_EVENT'
+    /**
+    * @typedef {String} SOLIDITY_EVENT_PROPERTY
+    */
+    export const SOLIDITY_EVENT_PROPERTY = 'SOLIDITY_EVENT_PROPERTY'
+
+    /**
+    * @typedef {Object} SolidityEventPropertySchema
+    * @property {Number} EventIndex  - the solidityEventId used by the EventStore
+    * @property {Number} EventPropertyIndex  - the property index used by the EventStore
+    * @property {String} Name - a string representing a key
+    * @property {String} Type - a string representing the type of the value
+    * @property {String} AddressValue - an address value
+    * @property {Number} UIntValue  - a uint value
+    * @property {String} StringValue - a string value
+    */
+    export const SolidityEventPropertySchema = {
+        EventIndex: 'BigNumber',
+        EventPropertyIndex: 'BigNumber',
+        Name: 'String',
+        Type: 'String',
+
```

Comment:
I get that you're separating values from event properties, but I'm just going to delete this if you don't